tools/src/golangci-lint/go.mod

```diff
@@ -1,12 +1,13 @@
 module local
 
-go 1.19
+go 1.20
 
-require github.com/golangci/golangci-lint v1.50.1
+require github.com/golangci/golangci-lint v1.51.2
```

Comment:
do not upgrade `golangci` for now？

Reply:
it gets killed, check out this run from a previous commit in this PR
https://github.com/envoyproxy/gateway/actions/runs/4368188859/jobs/7640450648#step:5:114

Reply:
my local machine got OOMed, after upgrade to go1.20.2 with no changes in golangci-lint.